src/screens/FeedScreen/Footer.js

```diff
@@ -103,12 +104,12 @@ const styles = StyleSheet.create({
 
 const handleTextCountColor = (totalVote) => {
   if (totalVote > 0) {
-    return '#00ADB5';
+    return COLORS.holytosca;
   }
   if (totalVote < 0) {
-    return '#FF2E63';
+    return COLORS.red;
   }
-  return '#C4C4C4';
+  return COLORS.gray1;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for maintainability and consistency. By using the `COLORS` object from the theme, it's easier to manage color schemes across the application. However, I would suggest adding a default case in the event that `totalVote` is not a number or is `NaN`. This will ensure that your function always returns a valid color.

```diff
 const handleTextCountColor = (totalVote) => {
   if (totalVote > 0) {
     return COLORS.holytosca;
   }
   if (totalVote < 0) {
     return COLORS.red;
   }
+  if (isNaN(totalVote)) {
+    console.warn(`Invalid totalVote: ${totalVote}`);
+    return COLORS.gray1; // Default color
+  }
   return COLORS.gray1;
 };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->